2019-01-11 11:58:27,190 ERROR: Exception on /user/clu [GET] [in /Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/blogApp/routes.py", line 106, in user
    return render_template('user.html', user=user, posts=posts)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
2019-01-11 11:58:27,198 ERROR: Exception on /user/clu [GET] [in /Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/blogApp/routes.py", line 106, in user
    return render_template('user.html', user=user, posts=posts)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
2019-01-11 11:58:27,435 ERROR: Exception on /user/clu [GET] [in /Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/blogApp/routes.py", line 106, in user
    return render_template('user.html', user=user, posts=posts)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/clu/Development/learn/learn-flask/simple-flask-heroku/venv/lib/python3.6/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: user.html
